ui/src/components/ProcessTree.scss

```
@@ -38,6 +38,20 @@
     line-height: 1;
+    margin: 0;
+    padding: 0;
+
+    .visually-hidden {
+      position: absolute;
+      width: 1px;
+      height: 1px;
+      padding: 0;
+      margin: -1px;
+      overflow: hidden;
+      clip: rect(0, 0, 0, 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)